src/index.d.ts

```diff
@@ -0,0 +1,15 @@
+declare module 'cli-diagram' {
+  class Diagram extends Array {
+    constructor(options?: object);
+
+    draw(): string;
+    height: number;
+    box(contents: string, options?: object): Diagram;
+    container(contents: string, options?: object): Diagram;
+    lines(lines: number | string[], options?: object): Diagram;
+    arrow(lines: number | string[], options?: object): Diagram;
```

Comment:
Arrow is actually `arrow(arrows: string[] | ({direction: string, label: string})[], options?: object): Diagram`

Though I don't expect a lot of people using the latter option.

Reply:
Sorted. :ok_hand: